tests/consequence_prediction/structural_variants/test_structural_pipeline.py

```diff
@@ -26,12 +26,12 @@ def run_pipeline(resource_name):
 
 def test_successful_run():
     assert sorted(run_pipeline('precise_genomic.xml.gz')) == sorted([
-        ['NC_000016.10:g.72059151_72063259del', 'ENSG00000140830', 'TXNL4B', 'intron_variant'],
+        ['NC_000016.10:g.72059151_72063259del', 'ENSG00000140830', 'TXNL4B', 'feature_truncation'],
         ['NC_000016.10:g.72059151_72063259del', 'ENSG00000257017', 'HP', 'stop_lost'],
-        ['NC_000016.10:g.72059151_72063259del', 'ENSG00000261701', 'HPR', 'coding_sequence_variant'],
-        ['NC_000001.11:g.25271785_25329047del', 'ENSG00000117616', 'RSRP1', 'intron_variant'],
+        ['NC_000016.10:g.72059151_72063259del', 'ENSG00000261701', 'HPR', 'feature_truncation'],
+        ['NC_000001.11:g.25271785_25329047del', 'ENSG00000117616', 'RSRP1', 'feature_truncation'],
         ['NC_000001.11:g.25271785_25329047del', 'ENSG00000187010', 'RHD', 'stop_lost'],
-        ['NC_000011.10:g.5226797_5226798insGCC', 'ENSG00000244734', 'HBB', 'coding_sequence_variant']
+        ['NC_000011.10:g.5226797_5226798insGCC', 'ENSG00000244734', 'HBB', 'feature_elongation']
```

Comment:
[Ensembl VEP 110](https://www.ensembl.info/2023/07/17/ensembl-110-has-been-released/) seems to have improved support for structural variants